packages/@aws-cdk/aws-ec2/lib/instance-types.ts

```diff
@@ -430,13 +430,6 @@ export enum InstanceClass {
    */
   C6GD = 'c6gd',
 
-  /**
-   * Compute optimized instances for high performance computing, 6th generation with Graviton2 processors
-   * and high network bandwidth capabilities
-   */
-  COMPUTE6_GRAVITON2_HIGH_NETWORK_BANDWITH = 'compute6-graviton2-high-network-banwidth',
```

Comment:
I think removing this would be a breaking change.

Reply:
Probably... but since `c6gbd` doesn't really exist (which `COMPUTE6_GRAVITON2_HIGH_NETWORK_BANDWITH` points to this one should be acceptable breaking change. Thoughts?

Reply:
wow I'm surprised nobody has found this yet. Looks like before the refactor it did have the correct value, but I guess nobody was using it. 

https://github.com/aws/aws-cdk/blob/b8d6cc79454b1533571e1c3f41b819523beac658/packages/@aws-cdk/aws-ec2/lib/instance-types.ts#L397

I'm fine with this change then, you can remove the breaking change message as well.